packages/flutter_tools/test/general.shard/flutter_platform_test.dart

```diff
@@ -57,6 +59,50 @@ void main() {
       ProcessManager: () => FakeProcessManager.any(),
     });
 
+    group('Observatory and DDS setup', () {
+      MockPlatform mockPlatform;
+      MockProcessManager mockProcessManager;
+      FlutterPlatform flutterPlatform;
+      final Map<Type, Generator> contextOverrides = <Type, Generator>{
+        Platform: () => mockPlatform,
+        ProcessManager: () => mockProcessManager,
+        FileSystem: () => fileSystem,
+      };
+
+      setUp(() {
+        mockPlatform = MockPlatform();
+        when(mockPlatform.isWindows).thenReturn(false);
+        mockProcessManager = MockProcessManager();
+        flutterPlatform = TestObservatoryFlutterPlatform();
+      });
+
+      Future<List<String>> captureCommand() async {
+        flutterPlatform.loadChannel('test1.dart', MockSuitePlatform());
+        when(mockProcessManager.start(
+            any,
+            environment: anyNamed('environment')),
+        ).thenAnswer((_) {
+          return Future<Process>.value(MockProcess());
+        });
+        await untilCalled(mockProcessManager.start(any, environment: anyNamed('environment')));
+        final VerificationResult toVerify = verify(mockProcessManager.start(
+          captureAny,
+          environment: anyNamed('environment'),
+        ));
+        expect(toVerify.captured, hasLength(1));
+        expect(toVerify.captured[0], isA<List<String>>());
```

Comment:
what is this testing?

Reply:
This was to test that I could return the command as a list. Not needed with the change to using FakeCommand